src/editor/index.tsx

```diff
@@ -43,6 +43,7 @@ interface EditorState {
   text: string;
   html: HtmlType;
   fullScreen: boolean;
+  composing: boolean;
```

Comment:
It's better to make composing as a normal variable, not as a state, because update state will cause re-render

Reply:
@sylingd 
Thank you. I fixed it.